main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
+var Version = "v0.0.0"
```

Comment:
technically, I can just do `var Version string` and be done with it. It _should_ be set by `make build` anyhow